news/_posts/2020-05-11-cms-security-summit.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "CMS Security Summit with Google"
+subtitle: "Security is very important for PrestaShop"
```

Comment:
Suggestion : this will be put inside a `<h4>` tag on homepage and a `<h3>` on the article page. Is this really a relevant title ? It seems a bit ... vague 😅?

Reply:
I used the same as last year. :sweat:

Reply:
Then 👍